### ISSUE TITLE
Transition To Workbench in May

### ISSUE THREAD

**Issue by zkamvar** [datacarpentry/R-ecology-lesson#841]
@datacarpentry/R-ecology-lesson-maintainers

As I hope you are already aware, we are rolling out the new lesson infrastructure, [The Carpentries Workbench](https://carpentries.github.io/workbench/), across all of The Carpentries official lessons in early May 2023. This means that all Data Carpentry, Library Carpentry, and Software Carpentry lesson repositories will be modified to adopt the new infrastructure in the coming days.

You can follow the transition of this lesson repository at https://github.com/carpentries/lesson-transition/issues/55.

The transition has already taken place for several lessons, and so far the process has been running quite smoothly. You should see the transition take place with minimal disruption, but there are a few things that it is important for Maintainers to be aware of. 

Here is what you can expect to happen next:

1. Any open pull requests on the repository will be closed with an automated message.
2. The repository will be set to read-only mode for a brief period while the transition occurs.
3. The new repository structure and lesson site layout will then be applied.
4. To avoid anyone accidentally pushing the old commit history back to the repository, after the transition Maintainers will need to delete and replace any existing forks and local clones they have of the lesson repository, and confirm that they have done so by replying to this issue.

I will reply here before and after the transition has taken place. If you have any questions in the meantime, please reach out to the Curriculum Team by tagging us here, e.g. @datacarpentry/core-team-curriculum.

If you would like to read more about the new lesson infrastructure and the modified repository structure you can expect post-transition, I recommend [the Infrastructure episode of the Maintainer Onboarding curriculum](https://carpentries.github.io/maintainer-onboarding/05-infrastructure.html) and [the Workbench Transition Guide](https://carpentries.github.io/workbench/transition-guide.html), which includes a side-by-side comparison of various elements of the old and new infrastructures.

**Comment by zkamvar** [comment]
We will transition this lesson on the first of May

**Comment by zkamvar** [comment]
I apologise, this will not be transitioned on the first of May. We still need to account for the former maintainers in the lesson release, which will be generated by @tobyhodges. 

This will be transformed on the 3rd of May

**Comment by zkamvar** [comment]
This lesson will be converted to use [The Carpentries Workbench][workbench]
To prevent accidental reversion of the changes, we are temporarily revoking
write access for all collaborators on this lesson:

 - [ ] @joongsup (push)
 - [ ] @jamesmbaazam (push)
 - [ ] @BResearcher101 (push)

If you no longer wish to have write access to this repository, you do not
need to do anything further.

1. What you can expect from the transition 📹: https://carpentries.github.io/workbench/beta-phase.html#beta
2. How to update your local clone 💻: https://carpentries.github.io/workbench/beta-phase.html#updating-clone
3. How to update (delete) your fork (if you have one) 📹: https://carpentries.github.io/workbench/faq.html#update-fork-from-styles

If you wish to regain write access, please re-clone the repository on your machine and
then comment here with `I am ready for write access :rocket:` and the
admin maintainer of this repository will restore your permissions.

If you have any questions, please reply here and tag @zkamvar

[workbench]: https://carpentries.github.io/workbench

**Comment by zkamvar** [comment]
The deed is done. The infrastructure takes a few minutes to bootstrap and cache the packages for the lesson build. Once the build is done, I will switch github pages to deploy from the `gh-pages` branch and you will have your workbench lesson. 

Thank you all for your enthusiasm and your patience!

**Comment by zkamvar** [comment]
Note: the root img folder will need to be deleted. This was an oversight during the transition.

**Comment by jamesmbaazam** [comment]
I am ready for write access :rocket:

**Comment by zkamvar** [comment]
Congratulations @jamesmbaazam, you now have write access!

**Comment by joongsup** [comment]
I'm ready for write access too. Thanks!

**Comment by ErinBecker** [comment]
Hi @BResearcher101 - You have not yet responded here to report that you completed the steps listed by @zkamvar above to regain your Maintainer access to this repository. The lesson infrastructure transition required a second Maintainer team to be temporarily created for all lessons, which can only be removed when all Maintainers have confirmed that they are ready to have their access restored. Maintenance of both teams is not sustainable for The Carpentries in the long run. At the beginning of November 2023 (note the deadline has been extended) we will remove access for Maintainers who have not taken the necessary action to continue working on the repository.

The Curriculum Team is truly grateful for everything you have already done as a Maintainer and we do not want to lose you! But we also understand that circumstances change and you may no longer have capacity to continue in your role. If you wish to retain your access and keep supporting the community, please follow the steps above then confirm that you are ready by commenting I am ready for write access 🚀 on this thread before the end of your day on Wednesday 1st November.

**Comment by ErinBecker** [comment]
Hi @BResearcher101 - You no longer have Maintainer access to this repository. Please see the comment above for details. If you would like to restore your access, please re-clone the repository on your machine and then comment here with I am ready for write access 🚀 and the admin maintainer of this repository will restore your permissions. If you have any questions, please don’t hesitate to get in touch [curriculum@carpentries.org](mailto:curriculum@carpentries.org)

To the remaining Maintainers for this lesson - the Curriculum Team will be prioritizing onboarding of new Maintainers for lessons that have lost Maintainers in our next round of onboarding. Please don’t hesitate to reach out if your lesson needs additional support in the meantime - [curriculum@carpentries.org](mailto:curriculum@carpentries.org).

I will close this issue now but will still be notified of any replies on the thread in future.
